src/app/styles/ng2-select.css

```diff
@@ -0,0 +1,273 @@
+
```

Comment:
Even better would be to configure the build to copy
```
./node_modules/ng2-select/components/css/ng2-select.css
```
to 
```
./dist/app/styles/ng2-select.css
```
and not bundle this file in
```
./src/app/styles/ng2-select.css
```
to keep our `src` folder clean with our files.